antarest/study/model.py

```diff
@@ -515,6 +517,7 @@ def transform_horizon_to_str(cls, val: str | int | None) -> Optional[str]:
 class StudyMetadataPatchDTO(AntaresBaseModel):
     name: Optional[str] = None
     author: Optional[str] = None
+    editor: Optional[str] = None
```

Comment:
We don't need to add it here: 
this object is used in the API to allow clients to update some study metadata, but they should not be able to modify the editor directly.
It's the application responsibility to update the editor depending on actions that are performed in the application (this is the goal of your other PR :) ).